README.md

```diff
@@ -2,26 +2,53 @@
 
 [![Actions Status](https://github.com/ruby/openssl/workflows/CI/badge.svg)](https://github.com/ruby/openssl/actions?workflow=CI)
 
+OpenSSL for Ruby provides access to SSL/TLS and general-purpose cryptography
+based on the OpenSSL library.
 
-OpenSSL provides SSL, TLS and general purpose cryptography. It wraps the
-OpenSSL library.
+OpenSSL for Ruby is sometimes referred to as "openssl" (in all lowercase) or
+"Ruby/OpenSSL" for disambiguation.
+
+## Compatibility and maintenance policy
+
+OpenSSL for Ruby is released as a RubyGems gem. At the same time, it is part of
+the standard library of Ruby. This is called a [default gem].
+
+Stable branches of OpenSSL for Ruby will receive security and bug fixes for the
+[lifecycle of their corresponding Ruby branch][Ruby Maintenance Branches].
```

Comment:
This? Not sure.

```
We only accept the security and bug fixes for the [Ruby maintenance branches in the life cycle][Ruby Maintenance Branches] for the stable branches of OpenSSL for Ruby.
```

Reply:
I think "branch/release will receive fixes/updates" is often used in similar contexts. I took a hint from the linked ruby-lang.org page which contains "[b]ranch receives general bug fixes and security fixes".

However, I figure I could simplify it more to something like the below. What do you think?

```suggestion
Each stable branch of OpenSSL for Ruby will remain supported as long as it is
included as a default gem in [supported Ruby branches][Ruby Maintenance Branches].
```

Reply:
OK. It looks nice. By the way, I am not familiar with the "Suggested change" feature on GitHub. (And I want to utilize it :)) The suggested change above is for the current line 17, but maybe you want to change the current lines 16 and 17.

Reply:
I wanted to do so, but I wasn't able to figure out how to change the lines. I think the original comment was attached to the line 17 only.